### PR TITLE
A mistake about the example description of the RegExp Character "+".

### PR DESCRIPTION
**Pull request by qwe1164915372**
### Description
A mistake about the example description of the RegExp Character "+".

### CODE REVIEW

File: files/zh-cn/web/javascript/guide/regular_expressions/quantifiers/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ slug: Web/JavaScript/Guide/Regular_expressions/Quantifiers
       <td>
         <p>
           将前一项“x”匹配 1
-          次或更多次。等价于{1,}。例如，/a+/匹配“candy”中的“a”和“caaaaaaandy”中的“a”。
+          次或更多次。等价于{1,}。例如，/a+/匹配“candy”中的“a”和“caaaaaaandy”中的“aaaaaaa”。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/guide/regular_expressions/quantifiers/index.md:41]: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/guide/regular_expressions/quantifiers/index.md?plain=1#L44

```suggestion
          次或更多次。等价于 <code>{1,}</code>。例如，<code>/a+/</code> 匹配“candy”中的“a”和“caaaaaaandy”中的所有“a”。
```

**Review by jasonren0403 — COMMENTED** [review]
Let's sync the EN content

File: files/zh-cn/web/javascript/guide/regular_expressions/quantifiers/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/javascript/guide/regular_expressions/quantifiers/index.md:41]: Hey @qwe1164915372. Do you forget to push the commit for this review? I saw you have marked this as resolved.

Comment (qwe1164915372) [reply files/zh-cn/web/javascript/guide/regular_expressions/quantifiers/index.md:41]: hello,I just check the branch "patch-1" of my "forked" and it’s what I modified.I want to know where to push, please give me some tips to complete this PR, thank you very much.
a1

Comment (yin1999) [reply files/zh-cn/web/javascript/guide/regular_expressions/quantifiers/index.md:41]: Just push commit to your remote branch [patch-1](https://github.com/qwe1164915372/translated-content/tree/patch-1). And all those commits would be included in this PR.

**Review by jasonren0403 — APPROVED** [review]
Thank you, and welcome to MDN 🎉

**Pull request merged**